### ISSUE TITLE
App not showing local titles in external

### ISSUE THREAD

**Issue by bbtora** [inorichi/tachiyomi#791]
I've got my hands on a tablet and decided to read manga on it but when I load local manga in directory, it doesn't show up. 
While I've assigned the downloads to external memory which are dling and showing just fine. Local ones aren't showing. 
sdcard>tachiyomi>local>title>.rar

Interesting thing is, the app recognises local manga stored in internal memory even when downloads are assigned to external. 
this is the path to external which is unusual in itself 
![img_20170513_234417_110](https://cloud.githubusercontent.com/assets/23010526/26028075/c2beecd0-3836-11e7-933f-a186113426c7.JPG)

Version: r1155

**Comment by inorichi** [comment]
Local manga doesn't use anything from the downloads directory. It tries to find multiple directories and merges them ([relevant code here](https://github.com/inorichi/tachiyomi/blob/master/app/src/main/java/eu/kanade/tachiyomi/util/DiskUtil.kt#L82)). Either more directories are added there (I don't have a phone with this issue so I can test it) or you wait until #662 is merged.

That path isn't unusual, it's the one the Storage Access Framework gives you, which is the only way to save data in an external SD since KitKat/Lollipop (with an Uri instead of a directory).

**Comment by bbtora** [comment]
Let me rephrase it. 

Local manga stored in external simply doesn't show up in catalogue. Only when I store them in internal does it work. @inorichi

**Comment by inorichi** [comment]
That's what I understood. Your external directory isn't detected in the code I linked you.
